packages/google_sign_in/google_sign_in_ios/darwin/google_sign_in_ios/Package.swift

```diff
@@ -0,0 +1,46 @@
+// swift-tools-version: 5.9
+
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import PackageDescription
+
+let package = Package(
+  name: "google_sign_in_ios",
+  platforms: [
+    .iOS("12.0"),
+    .macOS("10.15")
+  ],
+  products: [
+    .library(name: "google-sign-in-ios", targets: ["google_sign_in_ios"])
+  ],
+  dependencies: [
+    // AppAuth and GTMSessionFetcher are GoogleSignIn transitive dependencies.
+    // Depend on versions which define modules.
```

Comment:
This mirrors: https://github.com/flutter/packages/blob/f7b12561aeb6be92fc973441ae0f26cfaa8df8f4/packages/google_sign_in/google_sign_in_ios/darwin/google_sign_in_ios.podspec#L19-L23